src/frontend/komponenter/Fagsak/Personlinje/Behandlingsmeny/Behandlingsmeny.tsx

```diff
@@ -0,0 +1,83 @@
+import { RessursStatus } from '@navikt/familie-typer';
+import { Menyknapp } from 'nav-frontend-ikonknapper';
+import KnappBase from 'nav-frontend-knapper';
+import Popover, { PopoverOrientering } from 'nav-frontend-popover';
+import React, { useState } from 'react';
+import { useHistory } from 'react-router';
+import { useBehandling } from '../../../../context/BehandlingContext';
+import { BehandlingStatus } from '../../../../typer/behandling';
+import { IFagsak } from '../../../../typer/fagsak';
+import { hentAktivBehandlingPåFagsak } from '../../../../utils/fagsak';
+import EndreBehandlendeEnhet from './EndreBehandlendeEnhet/EndreBehandlendeEnhet';
+
+interface IProps {
+    fagsak: IFagsak;
+}
+
+const Behandlingsmeny: React.FC<IProps> = ({ fagsak }) => {
+    const aktivBehandling = hentAktivBehandlingPåFagsak(fagsak);
+    const skalViseOpprettBehandlingKnapp =
+        aktivBehandling === undefined ||
+        (aktivBehandling && aktivBehandling.status === BehandlingStatus.AVSLUTTET);
+
+    const history = useHistory();
+    const { åpenBehandling } = useBehandling();
+
+    const [anker, settAnker] = useState<HTMLElement | undefined>(undefined);
+
+    return (
+        <>
+            <Menyknapp
+                id={'behandlingsmeny-arialabel-knapp'}
+                mini={true}
+                onClick={(event: React.MouseEvent<HTMLButtonElement, MouseEvent>) =>
+                    settAnker(anker === undefined ? event.currentTarget : undefined)
```

Comment:
Nice. Er dette noe vi egentlig bør begynne å gjøre overalt?

Reply:
Nei, ikke med mindre man trenger å bruke ankeret til noe tenker jeg. Den kobler sammen knappen med popoveren.

Reply:
Ah sånn ja, for popoveren legger seg rundt ankeret. Skjønner :)